Source/WebCore/platform/graphics/cg/GraphicsContextCG.h

```diff
@@ -140,6 +151,8 @@ class WEBCORE_EXPORT GraphicsContextCG : public GraphicsContext {
     const RenderingMode m_renderingMode : 1; // NOLINT
     const bool m_isLayerCGContext : 1;
     mutable bool m_userToDeviceTransformKnownToBeIdentity : 1 { false };
+    // Flag for pending draws. Start with true because we do not know what commands have been scheduled to the context.
+    bool m_hasDrawn : 1 { true };
```

Comment:
Would prefer a name that indicates when this bit gets cleared, like `m_hasDrawnSinceLastCommit` or whatever.

Reply:
done.